console/src/components/WeatherItem/WeatherItem.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+import kind from '@enact/core/kind';
+import {Row, Column, Cell} from '@enact/ui/Layout';
+
+import css from './WeatherItem.less';
+import SunIcon from './icons/wi-day-sunny.svg';
+import Skinnable from '@enact/agate/Skinnable';
+import PropTypes from 'prop-types';
+
+const WeatherItemBase = kind({
+	name: 'WeatherItem',
+
+	styles: {
+		css,
+		className: 'weatherItem'
+	},
+	handlers: {},
+	computed: {
+		className: ({featured, styler}) => styler.append(featured ? css.featured : '')
+	},
+
+	render: ({label, description, high, low, ...rest}) => {
+		delete rest.featured;
+
+		return (
+			<Column {...rest} align="center">
+				<Cell className={css.item} shrink>
+					<Row align=" center" className={`${css.text} ${css.header}`}>
```

Comment:
How about instead of defining a new header style, we use `Divider` here and simplify the nesting?
```
<Cell className={css.item} component={Divider} shrink>
	{label}
</Cell>
```